src/commands/tags/getTagDiagnostics.ts

```diff
@@ -82,10 +82,15 @@ class TagVisitor implements jsonc.JSONVisitor {
                 this.addError(range, localize('tagNameTooLong', 'Tag name must be {0} characters or less.', max));
             }
 
+            if (/^\s*$/.test(property)) {
```

Comment:
nit: I think using something like `property.trim() === ''` would be a lil more readable than the regex but maybe that wouldn't have the exact same behavior